tests/mock_tables/dbconnector.py

```diff
@@ -50,7 +64,16 @@ def __init__(self, *args, **kwargs):
             raise ValueError("Invalid db")
```

Comment:
For single namespace test, can we just use your `dotabase_config.json`?

Reply:
Added a function to load database_config.json from local test directory. Using the local database_config.json in test_interfaces/test_hc_interfaces/test_pfc -interface related tests modified in this PR.